.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,10 @@
+vvvvvv Please remove this section before submitting vvvvvv
+
 ## Please verify the following:
 - [ ] Everything works on iOS/Android
 - [ ] `ignite-base` **ava** tests pass
```

Comment:
Yes.  We'll be using CI to enforce these rules automatically.  No need to put them in the PR template, imho.